tools/VscExtension/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "DocFXPreview",
-  "displayName": "DocFXPreview",
-  "description": "Preview Html and token tree generated by DocFX in Visual Studio Code",
-  "version": "0.0.24",
+  "name": "DocFX",
+  "displayName": "DocFX",
+  "description": "An amazing tool to make community users using and contributing to DocFX easier and more convenient, the main features includes DocFX Flavored Markdown previewing with default template or Docs template, generating token tree of markdown file",
```

Comment:
or: previewing ... , generating ...

---
In reply to: [131828463](https://github.com/dotnet/docfx/pull/1936#discussion_r131828463) [](ancestors = 131828463)